README.md

```diff
@@ -519,6 +519,33 @@ It is possible to specify flags to pass to `pg_ctl(1)` when performing the
 `restart` action, setting the `PGENV_RESTART_OPTIONS` array in the
 [configuration](#pgenv-config).
 
+### pgenv status
+
+Indicates whether an instance is already *running* or is *stopped*. This option
+is only meaningful when an instance is already in use. In case an instance is
```

Comment:
I would remove the "is only meaningful ..." sentence, is not clear what you mean: since the script always tells if the server is running or not, the command _is_ useful.